### ISSUE TITLE
Dependencies conflict with Virtus

### ISSUE THREAD

**Issue by greyblake** [mbj/vanguard#10]
I'd like to use it with the latest version of Virtus.

Gemfile:

``` ruby
gem 'vanguard'
gem 'virtus', '1.0.2'
```

But I got errors on bundle install:

```
Bundler could not find compatible versions for gem "ice_nine":
  In Gemfile:
    virtus (= 1.0.2) ruby depends on
      axiom-types (~> 0.1) ruby depends on
        ice_nine (~> 0.11.0) ruby

    vanguard (>= 0) ruby depends on
      adamantium (~> 0.0.8) ruby depends on
        ice_nine (0.8.0)
```

**Comment by ruanwz** [comment]
+1, please update the dependency.

**Comment by mbj** [comment]
I'll do so when I find the time.

**Comment by mikz** [comment]
Same issue here. @mbj Would you be ok with a pull request?

**Comment by mbj** [comment]
@mikz Yeah. Please do so.

**Comment by mikz** [comment]
@mbj Do you mind adding appraisals for testing different version combinations?

**Comment by mbj** [comment]
@mikz I generally do not see and advantage. We can discuss this but lets use two different PRs, one that fixes the initial issue and one 2nd where you show me the advantage of appraisals? This will reduce the time to release a fix.

**Comment by mbj** [comment]
@mikz Also vanguard is very badly specified, not even close to my other open sourced libraries quality. When I ever have time for it I plan to base vanguard on top of my [morpher](https://github.com/mbj/morpher) project. Vanguard will be a thiny DSL frontend. A single version bump should be releasable.

**Comment by mbj** [comment]
*Single dependency related version bump.

**Comment by mikz** [comment]
Ok. Will do it in next 24 hours.

**Comment by mbj** [comment]
I just released `v0.0.5` that should fix this conflict. Thx for PR by @mikz with the hard work.
